platform/security/platform-security-core-api/src/main/java/ddf/security/SystemHighAttributes.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.security;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * This code is experimental. While this interface is functional and tested, it may change or be
+ * removed in a future version of the library.
+ */
+public interface SystemHighAttributes {
+
+  /**
+   * @return an {@link Optional} of the system high attribute {@link Serializable} values, or {@link
+   *     Optional#empty()} if the attribute for the {@code attributeName} is not present
+   */
+  Optional<List<Serializable>> getValues(final String attributeName);
```

Comment:
Don't know if it is directly useful right now but it seems like this interface could use a method that returns all system high attributes. Maybe just the attribute names.

Reply:
To follow up after our discussion with @stustison and @lessarderic, we're going to keep just this one method for now and iterate in the future once we investigate differentiating between the user claims and the ism attributes defined in the users.attributes file.